[ACTP-631] add permissions handling to private action runner docs (#29141)
DataDog/documentation: content/en/actions/private_actions/use_private_actions.md

See [Connect a runner](#connect-a-runner) for more information on pairing your runner with a connection.

> ## Private action runner permissions

Reviewer: ```suggestion
## Manage access
```

Use [role-based access control (RBAC)][18] to control access to your private action runner. To see the list of permissions that apply to private action runner, see [Datadog Role Permissions][19].

> Set permissions on the runner to restrict its modifications or attachment of new connections to it. The granular permissions include **Viewer**, **Contributor**, and **Editor**. 

Reviewer: ```suggestion
You can set permissions on the runner to restrict modifications or prevent new connections from being attached. Available granular permissions include **Viewer**, **Contributor**, and **Editor**. 
```

> By default, only the creator of the runner receives **Editor** access. The creator can choose to grant access to additional users, service accounts, roles, or teams.

Reviewer: ```suggestion
By default, only the runner's creator has **Editor** access. The creator can grant access to additional users, service accounts, roles, or teams.
```

### Permission levels

Viewer
: Can view the runner and the connections attached to it

Contributor
: Can view and contribute to the runner by attaching new connections to it

Editor
: Can view, contribute (attach new connections), and edit the runner

### Set permissions on a runner

1. Navigate to the Edit page of the runner.

> 2. Go to the **Who Has Access?** section and click on the button **Edit access** .

Reviewer: ```suggestion
2. In the **Who Has Access?** section, click **Edit access**.
```

> 3. Select a user, a service account, a role or a team from the dropdown menu. Click **Add**. The principal you selected populates into the bottom of the dialog box.

Reviewer: ```suggestion
3. Select a user, service account, role, or team from the dropdown menu, then click **Add**. The selected principal appears at the bottom of the dialog box.
```

4. Next to the principal name, select your desired permission from the dropdown menu.

> 5. If you would like to remove access from a principal, in the dropdown for permissions choose **Remove access**.

Reviewer: ```suggestion
5. To remove access from a principal, select **Remove access** from the permissions dropdown menu.
```
